### PR TITLE
Use webpack for client

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/index.pug [hunk 1]
```diff
@@ -6,19 +6,9 @@ html(lang="en")
...
     title U.S. Tax Court Electronic Filing and Case Management System
-    link(rel="stylesheet" href="index.scss")
-    link(rel="stylesheet" href="../../node_modules/@fortawesome/fontawesome-svg-core/styles.css")
-    link(rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png")
-    link(rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png")
-    link(rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png")
-    link(rel="manifest" href="./site.webmanifest")
-    link(rel="mask-icon" href="./safari-pinned-tab.svg" color="#5bbad5")
-    meta(name="msapplication-TileColor" content="#da532c")
-    meta(name="theme-color" content="#ffffff")
```
Comment (kkoskelin) [inline web-client/src/index.pug:17]: but-why.gif

File: webpack.config.client.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackPugPlugin = require('html-webpack-pug-plugin');
+const webpack = require('webpack');
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        exclude: /node_modules/,
+        test: /\.(js|jsx)$/,
```
Comment (kkoskelin) [inline webpack.config.client.js:12]: `test: /\.jsx?$/,`

File: webpack.config.client.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        use: ['babel-loader'],
+      },
+      {
+        test: /\.(pdf|woff|woff2|ttf)$/,
```
Comment (kkoskelin) [inline webpack.config.client.js]: `test: /\.(pdf|woff2?|ttf)$/,`

**Review by rachaelparris — APPROVED** [review]
Nice!

File: webpack.config.client.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        use: ['file-loader'],
+      },
+      {
+        test: /\.(png|svg|jpg|jpeg|gif)$/i,
```
Comment (kkoskelin) [inline webpack.config.client.js]: `test: /\.(png|svg|jpe?g|gif)$/i,`
Noticing this one is case-insensitive (`/i`) -- should all the others be likewise?

File: webpack.config.client.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        use: ['pug-loader'],
+      },
+    ],
+  },
+  output: {
+    filename: `index.${Date.now()}.js`,
+    path: __dirname + '/dist',
+  },
+  plugins: [
+    new FaviconsWebpackPlugin('./web-client/src/favicon-32x32.png'),
```
Comment (kkoskelin) [inline webpack.config.client.js:38]: Does this do all the right "icon" things? sizes, platforms, etc?

**Review by kkoskelin — COMMENTED** [review]
The regex changes are just nitpicks, but I have more concerns about the number of `link` tags removed from the pug template.  Are those being handled in some way that isn't immediately obvious?

**Pull request merged**